src/init/features/composer/repo.ts

```diff
@@ -25,10 +25,6 @@ function extractRepoSlugFromURI(remoteUri: string): string | undefined {
   return match[1];
 }
 
-function generateConnectionId(stackId: string): string {
-  return `composer-${stackId}-conn`;
-}
-
```

Comment:
q: why do we remove this 🤔 . Do you think it's better to re-use stack id as the name of the connection id?

Reply:
I felt the user might be happy to see the stackId that they provided in the cloud build than appending something to it. So, I removed this. And if I have to use the connectionID in any other place I need to append `composer-${}-conn` this may lead to changing connectionID in multiple places(if required in future). To keep things simple I felt `stackId` is sufficient.

Reply:
sgtm. We can easily change this later if we change our mind.